generator/src/generators/forth.cr

```diff
@@ -8,9 +8,9 @@ class ForthGenerator < ExerciseGenerator
 
   def test_cases
     test_cases = [] of JSON::Any
-    JSON.parse(data)["cases"].each do |group|
```

Comment:
I think, as written, we'd want to keep both iterators to `each` since we're shoveling into a larger array (`test_cases`) and don't care about the returned array from `map`.

We might also be able to do some kinda `flat_map` like https://github.com/exercism/crystal/pull/128/files#diff-393e9e575152abac2ee842c029670674R10 but that can be done in some refactor or something if someone is inclined. Up to you.